p2p/src/msg.rs

```diff
@@ -325,11 +361,8 @@ pub struct Shake {
 
 impl Writeable for Shake {
 	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), ser::Error> {
-		ser_multiwrite!(
-			writer,
-			[write_u32, self.version],
-			[write_u32, self.capabilities.bits()]
-		);
+		self.version.write(writer)?;
+		ser_multiwrite!(writer, [write_u32, self.capabilities.bits()]);
```

Comment:
You can get rid of the ser_multiwrite! here, and just use writer.write_u32(self.capabilities.bits())

Reply:
:+1: